invenio_pidstore/models.py

```diff
@@ -39,6 +39,7 @@
 
 from .errors import PIDAlreadyExists, PIDDoesNotExistError, PIDInvalidAction, \
     PIDObjectAlreadyAssigned
+from flask_babelex import gettext as _
```

Comment:
It should be ``lazy_gettext``, but Flask-Admin is not happy about lazy strings so need to figure out a solution.